src/store/status_counts.ts

```diff
@@ -102,6 +143,34 @@ class StatusCountModule extends VuexModule {
   get fromProfile(): (filter: Filter) => number {
     return filter => this.hash(filter)['From Profile'];
   }
+
+  get passedTests(): (filter: Filter) => number {
+    return filter => this.hash(filter)['passedTests'];
+  }
+
+  get failedTests(): (filter: Filter) => number {
+    return filter => this.hash(filter)['failedTests'];
+  }
+
+  get failedOutOf(): (filter: Filter) => number {
+    return filter => this.hash(filter)['failedOutOf'];
+  }
+
+  get notApplicableTests(): (filter: Filter) => number {
+    return filter => this.hash(filter)['notApplicableTests'];
+  }
+
+  get notReviewedTests(): (filter: Filter) => number {
+    return filter => this.hash(filter)['notReviewedTests'];
+  }
+
+  get erroredTests(): (filter: Filter) => number {
+    return filter => this.hash(filter)['erroredTests'];
+  }
+
+  get erroredOutOf(): (filter: Filter) => number {
+    return filter => this.hash(filter)['erroredOutOf'];
+  }
```

Comment:
These functions are all exactly the same, can we just have a `filterTests() : (filter: Filter, type: String)` that just passes the type in?